env/GAEAC5.env

```
@@ -50,7 +50,7 @@ case ${step} in
 
     export APRUN="${APRUN_default}"
  ;;
- "waveinit" | "waveprep" | "wavepostsbs" | "wavepostbndpnt" | "wavepostpnt" | "wavepostbndpntbll")
+ "waveinit" | "waveprep" | "wavepostsbs" | "wavepostbndpnt" | "wavepostpnt" | "wavepostbndpntbll"| "wavestat" )
```

Add a space so it matches the rest of the line:
```suggestion
 "waveinit" | "waveprep" | "wavepostsbs" | "wavepostbndpnt" | "wavepostpnt" | "wavepostbndpntbll" | "wavestat" )
```